### ISSUE TITLE
Insert matplotlib plots

### ISSUE THREAD

**Issue by Vayel** [JelteF/PyLaTeX#35]
Would you be interested in a simple way to insert plots created with matplotlib? Just need to:

* Save the plot as `plot.pdf`
* Include this file
* Delete `plot.pdf`

Looks like:

```python
import matplotlib.pyplot as plt

x = [0, 1, 2, 3, 4, 5, 6]
y = [15, 2, 7, 1, 5, 6, 9]
    
plt.plot(x, y)
plt.savefig("./plot.pdf")
```

```tex
\documentclass{article}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage{graphicx}


\begin{document}

Some text.

\begin{figure}[htbp]
  \centering
  \includegraphics[width=\textwidth]{plot}
  \caption{SVG sexy image}
\end{figure}

Some other text.

\end{document}
```

**Comment by JelteF** [comment]
Yes, sure I would. It might be good to make it a subclass or wrapper of the Figure class. It you could give it the plt and it would do the savefig (to a unique filename) itself.

**Comment by JelteF** [comment]
Fixed by #42
